proxy_agent/Cargo.toml

```diff
@@ -59,3 +59,21 @@ features = [
   "Win32_System_ProcessStatus",
   "Win32_System_Kernel",
 ]
+
+[package.metadata.deb]
```

Comment:
do you need add cfg(not(windows)) to the section below or package.metadata.deb is implict linux based?

Reply:
No need since it only works at Linux build.